### ISSUE TITLE
Improve "Options" interface 

### ISSUE THREAD

**Issue by amosfolz** [VROOM-Project/vroom-frontend#37]
This is just a feature suggestion. 

If you select a location and then use Options to set it as a Start for a vehicle, you are unable to set that same location as the End. (to create a round trip) It only gives you the Del option. 

![image](https://user-images.githubusercontent.com/33728190/47971189-14c78300-e05d-11e8-8177-4140e8e8b269.png)

**Comment by jcoupey** [comment]
Good point, a vehicle with same start/end is only easy to create on the first location addition (or when cloning an existing vehicle).

I'll try to add this when I have time. For now a workaround using the interface would be to geocode twice the same place to assign it to both start and end.
